ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -854,26 +873,35 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
           <FlexRow>
             <FlexColumn>
               <FlexColumn  style={styles.researchPurposeRow}>
+                <FlexRow>
                 <CheckBox
                   data-test-id='researchPurpose-checkbox'
-                  style={{...styles.checkboxStyle, marginLeft: '0.6rem'}}
-                  label={this.researchPurposeLabel}
-                  labelStyle={styles.shortDescription}
-                  checked={this.primaryPurposeIsSelected}
-                  onChange={v => this.setState({
-                    selectResearchPurpose: v,
-                    showResearchPurpose: !this.state.showResearchPurpose})}/>
-                  {this.state.showResearchPurpose && <FlexColumn>
+                  manageOwnState={false}
+                  style={{...styles.checkboxStyle, marginLeft: '0.6rem', marginTop: '0.1rem'}}
+                  checked={this.researchPurposeCheck}
+                  onChange={v => this.onResearchPurposeChange(v)}/>
+                  <div style={{...styles.shortDescription, marginLeft: '-0.5rem'}}>
+                    <button style={{...styles.shortDescription, border: 'none'}} data-test-id='research-purpose-button'
+                            onClick={() => this.setState({showResearchPurpose: !this.state.showResearchPurpose})}>
+                      Research purpose
+                      {!this.state.showResearchPurpose &&
+                      <i className='pi pi-angle-right' style={{verticalAlign: 'middle'}}></i>}
+                      {this.state.showResearchPurpose &&
+                      <i className='pi pi-angle-down' style={{verticalAlign: 'middle'}}></i>}
```

Comment:
nit:  you could combine this with the tag above and use a ternary operator within the className